### PR TITLE
Update auth screen

### PR DESCRIPTION
**Pull request by dhruvanbhalara**
Added text field controller to hold entered text
Fix text field placeholder alignment
Added text capitaliztion
Added keyboard type

### CODE REVIEW

File: lib/Screens/Login/auth.dart [hunk 1]
```diff
@@ -59,12 +68,8 @@ class _AuthScreenState extends State<AuthScreen> {
       "themeColor": "",
       "colorHue": "",
       "lastLogin": "",
-      "accountCreatedOn": DateTime.now()
-          .toUtc()
-          .add(Duration(hours: 5, minutes: 30))
-          .toString()
```
Comment (Sangwan5688) [inline lib/Screens/Login/auth.dart:65]: I think these were according to flutter formatting. Changing these might fail flutter tests.

File: lib/Screens/Login/auth.dart [hunk 2]
```diff
@@ -149,8 +153,7 @@ class _AuthScreenState extends State<AuthScreen> {
                     Padding(
                       padding: const EdgeInsets.only(left: 10, right: 10),
                       child: Container(
-                        padding: EdgeInsets.only(
-                            top: 5, bottom: 5, left: 10, right: 10),
```
Comment (Sangwan5688) [inline lib/Screens/Login/auth.dart:153]: Same goes here.

File: lib/Screens/Login/auth.dart [hunk 3]
```diff
@@ -165,10 +168,12 @@ class _AuthScreenState extends State<AuthScreen> {
                         ),
                         child: TextField(
                             controller: controller,
+                            textAlignVertical: TextAlignVertical.center,
+                            textCapitalization: TextCapitalization.sentences,
+                            keyboardType: TextInputType.name,
                             decoration: InputDecoration(
                               focusedBorder: UnderlineInputBorder(
-                                borderSide: BorderSide(
-                                    width: 1.5, color: Colors.transparent),
```
Comment (Sangwan5688) [inline lib/Screens/Login/auth.dart:171]: Might break flutter format

**Review by Sangwan5688 — COMMENTED** [review]
Everything looks fine just make sure the formatting is right. As it will break the build.

File: lib/Screens/Login/auth.dart [hunk 4]
```diff
@@ -16,11 +17,19 @@ class _AuthScreenState extends State<AuthScreen> {
...
   @override
   void initState() {
     main();
     super.initState();
+    controller = TextEditingController();
+  }
+
+  @override
+  void dispose() {
+    controller.dispose();
```
Comment (Sangwan5688) [inline lib/Screens/Login/auth.dart:31]: ════════ Exception caught by widgets library ════════
The following NoSuchMethodError was thrown while finalizing the widget tree:
The method 'dispose' was called on null.
Receiver: null
Tried calling: dispose()

**Comment by dhruvanbhalara** [comment]
@Sangwan5688 Updated file formatting according to dart

File: lib/Screens/Library/nowplaying.dart [hunk 5]
```diff
@@ -86,55 +87,59 @@ class _NowPlayingState extends State<NowPlaying> {
...
-                                                                              ),
-                                                                          placeholder: (BuildContext context, _) =>
-                                                                              Image(
-                                                                                image: AssetImage('assets/cover.jpg'),
-                                                                              ),
-                                                                          imageUrl: mediaItem
-                                                                              .artUri
-                                                                              .toString())
-                                                            
-                                                        ),
```
Comment (Sangwan5688) [inline lib/Screens/Library/nowplaying.dart:137]: This has already been formatted in the latest commit. I guess you were using an old one. Anyway, no issues.

**Comment by Sangwan5688** [comment]
Thanks for the PR, looking forward for more in future :)

**Pull request merged**